improver/nowcasting/forecasting.py

```diff
@@ -359,7 +361,8 @@ def grid_spacing(coord):
 
         # perform advection and create output cube
         advected_data = self._advect_field(cube.data, grid_vel_x, grid_vel_y,
-                                           timestep.total_seconds())
+                                           int(round(timestep.total_seconds()))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [round built-in referenced](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4935535)